pyee/_trio.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+
+from contextlib import asynccontextmanager
+import trio
+from pyee._base import BaseEventEmitter, PyeeException
+
+__all__ = ['TrioEventEmitter']
+
+
+class TrioEventEmitter(BaseEventEmitter):
+    """An even emitter class which can run trio coroutines in a trio nursery.
```

Comment:
```suggestion
    """An even emitter class which can run trio tasks in a trio nursery.
```

IIUC, tasks are the idea: https://trio.readthedocs.io/en/stable/design.html#basic-principles.

Reply:
Good call. Definitely want to use the UBIQUITOUS LANGUAGE here!